tests/unittest_odgovlt.py

```diff
@@ -0,0 +1,57 @@
+from ckanext.harvest.harvesters.ckanharvester import CKANHarvester
+from ckanext.harvest.harvesters.ckanharvester import DatetimeEncoder
+from mock import patch
+from sqlalchemy import MetaData
+from sqlalchemy.engine.base import Engine
+from datetime import datetime
+
+ckan_object = CKANHarvester()
+datetime_object = DatetimeEncoder()
```

Comment:
Never execute code in module scope. Move those calls to test functions. When you execute something in the module scope it creates so called import side effects. And those import side effects causes bunch of different issues.

All initialization should go to test functions or fixtures.